CONTRIBUTING.md

```diff
@@ -578,7 +578,9 @@ See also:
 The tests should never leak goroutines.
 
 Use the term `ConcurrentSafe` in the test name when it aims to verify the
-absence of race conditions.
+absence of race conditions. The top-level tests with this term will be run
+100 times in the `test-concurrent-safe` CI job to increase the chance of
+catching concurrency issues. (this does not apply to subtests)
```

Comment:
What do you mean that it does not apply to subtests?

Reply:
This is the limitation of the `go test`. With `-run=ConcurrentSafe` args, `go test` won't run a subtest like `Foo/ConcurrentSafe`. It can only match the root test like `FooConcurrentSafe`.

Though we can use `-run=/ConcurrentSafe` to match subtests, it would evaluate all root tests first to determine the subtests, which means running all root tests. This would significantly increase the testing time.

So, this rule does not apply to subtests when the term is not in their root name.

Reply:
I first read this as "a test named `ConcurrentSafeXXX/subtest` will not run. Maybe it could be clarified?

Reply:
Updated